kernel/modules/Ramdisk.cpp

```diff
@@ -45,6 +45,10 @@ void ramdisk_load(Module *module)
 
             fshandle_destroy(handle);
         }
+        else if (block.name[strlen(block.name) - 1] != '/')
+        {
+            filesystem_mklink_for_tar(file_path, (Path*) block.linkname);
```

Comment:
You need to use path_create and path_destroy for converting a char* to a Path*

Reply:
I think it's fixed now